inc/class-s3-destination.php

```diff
@@ -165,6 +165,9 @@ public static function options()
                     'label' => __('Scaleway: AMS', 'backwpup'),
                     'endpoint' => 'https://s3.nl-ams.scw.cloud',
                 ],
+                'custom' => [
+                    'label' => __('Custom S3 destination', 'backwpup'),
+                ],
```

Comment:
What's the point of adding this custom destination? A custom destination is already supported in the `::fromJobId()` method ([line 207+](https://github.com/inpsyde/backwpup/blob/4ea1f40efcc7d875214952838a34ca3d0fd5deb6/inc/class-s3-destination.php#L207)).